app/css/ui-kit/filters.css

```diff
@@ -15,4 +15,9 @@
     .filter-lightGold {
         filter: var(--lightGold-filter);
     }
+
+    .filter-shadow-huge {
+        -webkit-filter: drop-shadow(0px 4px 128px rgba(79, 114, 205, 0.8));
+        filter: drop-shadow(0px 4px 128px rgba(79, 114, 205, 0.8));
```

Comment:
You can do this via tailwind, which removes the need for the extra webkit variant: https://tailwindcss.com/docs/drop-shadow#arbitrary-values

Reply:
@dem4ron